src/site/content/en/reliable/http-cache/index.md

```diff
@@ -1,52 +1,77 @@
 ---
 layout: post
-title: 'The HTTP cache: your first line of defense'
+title: 'The HTTP Cache: Avoiding unnecessary network requests with `Cache-Control` and `ETag`'
 authors:
   - jeffposnick
+  - ilyagrigorik
```

Comment:
Feel free to add your name here, too, Kayce!

Reply:
Thanks, I thought about it, but it bugs me when there's a long list of authors on a page.